### DISCUSSION TITLE
Possibility to import EEVEE csv to teslamate/grafana to cover the lost of data?

### DISCUSSION THREAD

**Discussion by Toasterme** [teslamate-org/teslamate#2547] [Q&A]
Hi guys.
I recently discovered a big hole in my data from January to March 2022 because of the login token expiration.  So Teslamate was unable to connect to get the data and i do not check my data often.   Yesterday i wanted to check how does the winter affect my car and saw that and fixed it with a new generated token...   Btw a email or a notice from my own Teslamate will be great for the next time something like that happen?  Or is it already available ? (notification on bad login)

I have **[EEVEE](https://apps.apple.com/us/app/eevee-for-tesla-bmw-ford/id1473252719)** also on my phone, with some data since i installed it (195 days to be exact).  So i've extracted a .**CSV** of it.  Is there a way to consolidate the EEVEE data to Teslamate data?

Data look like this in the .csv:

> Location,"Start time","End time","KWh added","KWh added dayrate","KWh added nightrate"
> Home,"2022-03-20 06:01","2022-03-20 07:45",13,13,0
> Home,"2022-03-16 06:00","2022-03-16 07:49",13.54,13.54,0
> Home,"2022-03-12 08:00","2022-03-12 10:04",15.43,15.43,0
> Home,"2022-03-09 08:00","2022-03-09 09:25",9.93,9.93,0
> Home,"2022-03-07 08:00","2022-03-07 08:58",7.14,7.14,0

**Comment by Hotanya** [comment]
Hey @Toasterme, did you ever find a solution for this? Thanks.

**Reply by jlestel** [reply]
Hi @Hotanya @Toasterme 

Did you find a way to do it?

**Reply by Toasterme** [reply]
@Hotanya  @jlestel nope! never, and i guess i never will because EEVEE will be paid subs now, and i won't have access to it anymore.
